### PR TITLE
Use IMAGE_REPO as default base for our artifacts

### PR DESCRIPTION
**Pull request by TerryHowe**

### CODE REVIEW

File: projects/kubernetes/kubernetes/Makefile [hunk 1]
```diff
@@ -11,20 +11,20 @@ CLONE_URL=https://github.com/$(COMPONENT).git
 AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
 AWS_REGION?=us-west-2
 
-BASE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
+KUBE_BASE_REPO?=$(IMAGE_REPO)
```
Comment (vivek-koppuru) [inline projects/kubernetes/kubernetes/Makefile:14]: Does this work if image repo is defined below instead of above?

Comment (TerryHowe) [reply projects/kubernetes/kubernetes/Makefile:18]: Good call.Fixed

**Review by bnrjee — APPROVED** [review]

**Review by bnrjee — APPROVED** [review]

**Review by vivek-koppuru — APPROVED** [review]
/lgtm

**Review by vivek-koppuru — APPROVED** [review]
/lgtm
/approve

**Pull request merged**